packages/agoric-cli/src/lib/rpc.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable @jessie.js/no-nested-await */
+// @ts-check
+/* global Buffer, process */
+
+/**
+ * @template K, V
+ * @typedef {[key: K, val: V]} Entry<K,V>
+ */
+
+// TODO get non-local from config file
+export const networks = {
+  local: { rpc: 'http://0.0.0.0:26657', chainId: 'agoric' },
+  xnet: { rpc: 'https://xnet.rpc.agoric.net:443', chainId: 'agoricxnet-13' },
+  ollinet: {
+    rpc: 'https://ollinet.rpc.agoric.net:443',
+    chainId: 'agoricollinet-30',
+  },
+};
```

Comment:
i'm ok with the "pass in the instagoric name" for the netname, but can we make chainid auto discovered?

e.g.

```sh
% curl -Ss https://ollinet.agoric.net/network-config | jq -r .chainName
agoricollinet-30
```

Reply:
I kinda like the `const net = networks[opts.net || 'local'] || JSON.parse(opts.net);` idea too.

Reply:
https://github.com/Agoric/agoric-sdk/pull/6176/commits/04f0e2c4ba2ecf07bb248759f140ab2bf057473a